tests/unit/wc-payment-api/test-class-wc-payments-api-client.php

```diff
@@ -1341,4 +1409,15 @@ private function set_http_mock_response( $status_code, $body = [], $headers = []
 				)
 			);
 	}
+
+	/**
+	 * Set up mock for no subscriptions for order.
+	 */
+	private function mock_no_subscriptions_for_order() {
```

Comment:
Let's omit the method introduction and use the closure injection right in the place.